pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,67 @@
+title: Concrete Action: Taking a Stand with Terminology and Recruitment
```

Comment:
How about "#BlackLivesMatter: Taking a Stand with Terminology and Recruitment"? Right now the title doesn't really make it clear what this post is about.